packages/ampath-esm-clinic-dashboard-hiv-lists/src/reports/cross-border-report/cross-border-report.component.test.tsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import {
+  cleanup,
+  fireEvent,
+  render,
+  RenderResult,
+  screen,
+} from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import CrossBorderReport from "./cross-border-report.component";
+import { BrowserRouter } from "react-router-dom";
+import { mockCrossBorderReportData } from "../../../__mocks__/cross-border-data.mock";
+import { amrsFetch } from "../../amrs-fetch/amrsFetch";
+
+const mockUseHistoryPush = jest.fn();
+const mockAmrsFetch = amrsFetch as jest.Mock;
+
+jest.mock("react-router-dom", () => ({
+  ...jest.requireActual("react-router-dom"),
+  useHistory: () => ({ push: mockUseHistoryPush }),
+  useLocation: jest.fn(),
+}));
+
+jest.mock("../../amrs-fetch/amrsFetch", () => ({
+  amrsFetch: jest.fn(),
+}));
+
+
+beforeEach(() => {
+  mockAmrsFetch.mockReset();
+});
+
+beforeEach(() => {
+  render(
+    <BrowserRouter>
+      <CrossBorderReport />
+    </BrowserRouter>
+  );
+});
+
+afterEach(cleanup);

```

Comment:
Just as an aside, Pro-tip: no need to [cleanup](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-cleanup).